cis/schema.json

```diff
@@ -29,54 +29,54 @@
     "properties": {
         "user_id": {
             "type": "string",
-            "description": "(Authoritative) The user ID as defined by the underlying IdP. Profile consumers should not try to interpret this value in any way, but may use it as a stable identifier for this user as identified by this IDP."
+            "description": "The unique user identifier. Profile consumers should not try to interpret this value in any way."
         },
         "timezone": {
             "type": "string",
-            "description": "(Informational) Timezone name for this user's typical location, e.g., 'Europe/London'."
+            "description": "Preferred timezone name for this user's location, e.g., 'Europe/London'."
         },
         "active": {
             "type": "boolean",
-            "description": "(Authoritative) If false, all access for this user should be denied. Profile consumers MUST check this value and disregard the remainder of the profile if false."
+            "description": "If false, all access for this user should be denied. Profile consumers MUST check this value and disregard the remainder of the profile if false."
```

Comment:
when querying the CIS API in the future - the point is that it is stored that way in the identity vault (ie the database of user profiles) thus anything that may read the profile would get "false" returned. Right now, this is only "internal" CIS code. Auth0 will return an HTTP error code.